root/main/ConfirmSeed.js

```diff
@@ -29,6 +29,17 @@ const ConfirmSeed = ({
 }: Props): React$Element<typeof Layout> => {
   const $c = React.useContext(SanitizedCatalystContext);
   const title = l('Confirm Form Submission');
+
+  /*
+   * Automatically submit the form if /release/add (where POST requests are
+   * used to seed fields rather than to create edits) was requested and the
+   * client indicated that they want the form to be skipped. See MBS-13225.
+   */
+  const url = new URL($c.req.uri);
+  const autoSubmit =
+    url.pathname === '/release/add' &&
+    url.searchParams.get('skip_confirmation') === '1';
```

Comment:
I'm requiring this parameter to make the new behavior opt-in, but I'm not sure whether there are actually any clients that try to click through the confirmation page. Let me know if you think it's safe to do this unconditionally.

Reply:
Opt-in for now seems safer at least, and doesn't cost much to support.  We can always remove it later if it's not needed.